lib/core/test/data/Cardano/Wallet/Api/ApiAddressData.json

```diff
@@ -0,0 +1,347 @@
+                                    "script_vkh1qqfs5ys3r59qqqqkrcwqxqqfpywqursyrv8qqqsnrsyqs3hg7wf",
+                                    "script_vkh1z5dqjqshq58sgqsuqsfqqqg8qv9pgqcnq5rpw8qxrcp3u3vx264"
+                                ]
+                            },
+                            {
+                                "some": {
+                                    "from": [
+                                        "script_vkh1zszs7yg6qcq3y8c9zqgs69cdrvrs5p3qp5p379qlzvsqgrh63lr",
+                                        "script_vkh1qypqjyq5qszjqzs7zsgsgzqpqqsp2pqtpg83vzsrz5r3y8vwnh9",
+                                        "script_vkh1pc8scgq0qcxs29gyzqsq2xsczc0psrsdzsx36xc7z5tqu926wps",
+                                        "script_vkh1pvxszyqnq5rqczg3qg932gqsrvrpyyclr5f3yyqmqcfpkcfrnzl",
+                                        "script_vkh1q5vzqzguqgp3vpgsqyrpy8q7zsfpzpgyzygp7zqnzuwsupu3s3t"
+                                    ],
+                                    "at_least": 2
+                                }
+                            },
+                            {
+                                "any": [
+                                    "script_vkh1qgg3zpgwr50qyrctqsp368g0zsppzqsgq5y3cpcfpsrqqwmmrf6",
+                                    "script_vkh1ps0pjyquqqxqvzcpqug378s4zczp69swzq2qjzfqqutpjqxnxq3",
+                                    "script_vkh1ruvpg8suqgv3c9qmzggpkzq0zur3qrgmrsv3jrg9pszpg7lnjjr",
+                                    "script_vkh1pggpjycqrgxjqpfqpsyqspclz5v3v8chzcd3g9qxpv93ugmqjwu",
+                                    "script_vkh1pstpgzchqcqqvqssr5fq7ysppuz3cgqrrg93src3rqvq76w26nn",
+                                    "script_vkh1p5xqyzsnry9qxrcaqsspk8qpqvxsqx3qzqvsyxslpgyssvm6a8c",
+                                    "script_vkh1pvwqwzc5qgpsyyqjzyxq7rqgrqvswxcmpgwsgpctzqfqxq960hu"
+                                ]
+                            },
+                            {
+                                "all": [
+                                    "script_vkh1pc2sx9q2rvrpspglqvfpuqqaruvsxxctzsszqpsrqyspgcly6ak",
+                                    "script_vkh1psrqkrcupvys2pglzg83xrcer5w3yzqlqcppkqqdrqvs2uvepvw",
+                                    "script_vkh1qsypgrgurgd3qpqzzuz3y8c2rygqc9cvzyz3y9c8yqvpgq4p647",
+                                    "script_vkh1q5v35rqqp5xpqqcprqvssrclp5zq6psnzydsx8qrqurqg2hpldc"
+                                ]
+                            }
+                        ],
+                        "at_least": 3
+                    }
+                }
+            }
+        },
+        {
+            "payment": {
+                "pub_key": "addr_vk1v6835v2g6yx4xqgyzu3x785snq4yks2t0e8mgkldxeyhmx57ksvshkua8r"
+            },
+            "stake": {
+                "pub_key": "addr_vk1ney9mxfedm5gas7x85rdzdz5npgcv5hqpm4au99jr3gumdjf66nqmk5t8r"
+            }
```

Comment:
It seems inconsistent with other primitives that there's an extra `pub_key` field here. What about simply:

```
{ "payment": "addr_vk1v6835v2g6yx4xqgyzu3x785snq4yks2t0e8mgkldxeyhmx57ksvshkua8r" }
```

and

```
{ "stake": "addr_vk1ney9mxfedm5gas7x85rdzdz5npgcv5hqpm4au99jr3gumdjf66nqmk5t8r" }
```

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/595a91aa4d4cc2fe1557d7564d32d0dc23daa3ef